packages/terra-paginator/tests/wdio/paginator-spec.js

```diff
@@ -1,72 +1,128 @@
 Terra.describeViewports('Paginator', ['tiny', 'medium', 'large'], () => {
   describe('Paginator', () => {
-    before(() => browser.url('/#/raw/tests/terra-paginator/paginator/paginator'));
-
-    Terra.it.validatesElement();
+    it('default', () => {
+      browser.url('/raw/tests/terra-paginator/paginator/paginator');
+      Terra.validates.element();
```

Comment:
Can you give names for all the new screenshots (wherever missing), as name will be a required argument in WDIO-6.

Reply:
Let's hold off on this for now as this PR is old and I'd like to get it in at this point.  If we want, we could do this on a separate PR.